src/main/java/com/smartcar/sdk/Vehicle.java

```diff
@@ -126,6 +130,48 @@ public String vin() throws SmartcarException {
     return this.call("permissions", "GET", null, ApplicationPermissions.class).getData().getPermissions();
   }
 
+  /**
+   * Checks if permissions granted to a vehicle contain the specified permission.
+   *
+   * @param permission Check if the vehicle has this permission granted
```

Comment:
the description is off, should explain what is the param!

Reply:
Sydney and I discussed it and thought this was the best description for the parameter, it is the same description in all SDKs.

Reply:
We decided to change it to just "Permissions to check" on all SDKs to keep it simple and just explain what is the param.